test/blackbox-tests/test-cases/coq/no-stdlib.t/run.t

```diff
@@ -0,0 +1,13 @@
+Test that when `(stdlib no)` is provided, the standard library is not bound to `Coq`
+and the prelude is not imported
+
+  $ dune build
```

Comment:
Maybe use `dune build --verbose` ? IMHO it is a good idea to guard here against silent flag changes.

Also let's add 2 tests where the boot library is in scope, one for yes and the other for now. You can just copy from the existing tests for boot copmosition.

Reply:
I tried `--verbose` but I don't think that is going to work, because it's output is not deterministic:
```
-|  Running[0]: /usr/bin/ocamlc.opt -config > /tmp/build_01aa54_dune/dune_143239_output
+|  Running[0]: /usr/bin/ocamlc.opt -config > /tmp/build_784671_dune/dune_e90675_output
```

Reply:
Thanks! Maybe you can try -j1 and that will make the build deterministic? I'm unsure how this is handled in other setups, maybe instead of `--display=verbose` `--display=short` would suffice here?

Or we can just test the build of a single file, like `dune build --verbose foo.vo`, I think that should be actually enough.

Reply:
Don't put `verbose` in tests it doesn't work.

Reply:
Yeah, I don't think that any of the verbose options are going to work. The issue is not so much with parallelism but with temporary directories.

Reply:
Mmm, I see other tests are using `dune rules` to check this? But anyways this seems like a PITA so let's use `--display=short` instead for a single file, and check that nothing else is build when `(stdlib no)`

Reply:
Done.

Reply:
Thanks!